src/omotes_rest/rest_interface.py

```diff
@@ -165,6 +166,14 @@ def get_workflows_jsonforms_format(self) -> list:
                         jsonforms_schema["format"] = "date-time"
                         if _parameter.default:
                             jsonforms_schema["default"] = _parameter.default.isoformat()
+                    elif isinstance(_parameter, DurationParameter):
```

Comment:
Question: On line 215 (in the function `submit_job`) the `input_params_dict` is taken straight from the `JobInput` dataclass (which is generated by the REST api). Where is this `input_params_dict` converted to Python objects? e.g. where is the isoformatted string converted to a Python `datetime.datetime` and where will the total_seconds received from the frontend converted to a `datetime.timedelta`?

Reply:
That will happen here:  
https://github.com/Project-OMOTES/omotes-sdk-python/blob/0270cc5538215243e8dca012a2b0f3ceeb599502/src/omotes_sdk/omotes_interface.py#L361  
which will call `to_pb_value` for each `WorkfloParameter` child instance.

Is that what you mean?

Reply:
Mark and I found that the conversion from JSONForms format to omotes-sdk-python params_dict format is missing. Datetime and duration should be converted from their string & number representations to `datetime.datetime` and `datetime.timedelta` objects before being submitted with `job_submission()`. Fixing in this issue https://github.com/Project-OMOTES/omotes-rest/issues/30